### PR TITLE
Refacture

### PR DESCRIPTION
**Pull request by nikstaaar**

### CODE REVIEW

File: src/pages/cocktails.js [hunk 1]
```diff
@@ -25,9 +40,22 @@ export default function CocktailPage() {
 			</Head>
 			<>
```
Comment (mbosselmann) [inline src/pages/cocktails.js]: Fragments are not needed here. :)

File: src/pages/bar.js [hunk 2]
```diff
@@ -1,107 +1,40 @@
...
-							setFilteredIngredients(ingredients);
-						}}
-					>
-						Done
-					</Button>
-				)}
-			</>
+			<IngredientGrid>
+				{!savedIngredients && <>There are no Ingredients in your Bar</>}
+				<Link href="/select">Add more Ingredients</Link>
```
Comment (mbosselmann) [inline src/pages/bar.js]: A anchor element is not allowed inside a `ul` element. [See caniinclude for further advice](https://caninclude.glitch.me/caninclude?child=a&parent=ul).

File: src/pages/cocktails.js [hunk 3]
```diff
@@ -25,9 +40,22 @@ export default function CocktailPage() {
...
 				<ul>
```
Comment (mbosselmann) [inline src/pages/cocktails.js]: `IngredientGrid` is already a `ul` element. You can remove this `ul` as you don't need it anymore.

File: src/pages/select.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+import {useState} from 'react';
+import {useEffect} from 'react';
+
+import Layout from '../components/Layout';
+import {IngredientCard, IngredientGrid} from '../components/styled/IngredientCard.styled';
+import ingredientsStore from '../hooks/ingredientsStore';
+
+export default function BarPage() {
+	const {data: session} = useSession();
+	//const ingredients = ingredientsStore(state => state.ingredients);
```
Comment (mbosselmann) [inline src/pages/select.js:12]: No outcommented code should be left in your PR. :)

File: src/pages/select.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+								}}
+							>
+								add
+							</button>
+						</IngredientCard>
+					);
+				})}
+			</IngredientGrid>
+
+			<button onClick={update}>save</button>
```
Comment (mbosselmann) [inline src/pages/select.js:54]: You should the button type here. Per default this one is a submit button. But you don't want to have a submit event here or? Add `type="button"` and it should be alright.

**Review by mbosselmann — COMMENTED** [review]
Hi Niklas,
great job! 🚀  I left some remarks for you to consider. 👀 Feel free to ask me if you have questions.

**Comment by nikstaaar** [comment]
> Hi Niklas, great job! 🚀 I left some remarks for you to consider. 👀 Feel free to ask me if you have questions.

thank you for your feedback. I have implemented your suggestions.

**Pull request merged**